docs/application/dotnet/guides/user-interface/nui/ui-components.md

```diff
@@ -11,28 +11,33 @@ The following table lists the available UI components:
-| [CheckBox](./nui-components/CheckBox.md) | A CheckBox that can set selected or unselected status when user selects it. | `CheckBox`                |
-| [ImageView](./imageview.md)          | An image view is a class for displaying an image resource.   | `ImageView`                   |
-| [Loading](./nui-components/Loading.md) | A loading is used to give information about the ongoing operations. | `Loading`                |
-| [LottieAnimationView](./nui-components/LottieAnimationView.md) | A LottieAnimationView is used to show a vector animation. | `LottieAnimationView`                |
-| [Notification](./nui-components/Notification.md) | A notification is used to pop-up a notification window with a content view. | `Notification`                |
-| [Pagination](./nui-components/Pagination.md) | A Pagination is used to show the number of pages available and the currently active page. | `Pagination`                |
-| [Popup](./nui-components/Popup.md) | A popup is used as a popup window. | `Popup`                |
-| [Progress](./nui-components/Progress.md) | A progress is used to show the ongoing status using a long narrow bar. | `Progress`                |
-| [RadioButton](./nui-components/RadioButton.md) | A RadioButton that can set selected or unselected status when user selects it. | `RadioButton`                |
-| [ScrollableBase](./nui-components/ScrollableBase.md) | A ScrollableBase that can support scrolling. | `ScrollableBase`                   |
-| [Slider](./nui-components/Slider.md) | A slider that indicates a modifiable value within a specific range. | `Slider`                   |
-| [Switch](./nui-components/Switch.md) | A switch that can be used as a selector. | `Switch`                |
-| [Tab](./nui-components/Tab.md) | A tab is easy to explore and switch between different views. | `Tab`                |
-| [TableView](./tableview.md)          | A table view that can align child actors in a grid like layout. | `TableView`             |
-| [Text](./text.md)                    | A text that renders a short text string and provides a editable text. | `TextLabel`, `TextField`, `TextEditor`  |
-| [Toast](./nui-components/Toast.md) | A toast is used to provide simple messages. | `Toast`                |
-| [VideoView](./videoview.md)          | A video view that controls and displays video playback. | `VideoView`                 |
-| [WebView](./webview.md)          | A web view for showing web content. | `WebView`                 |
+| [AlertDialog](./nui-components/AlertDialog.md)  | AlertDialog class shows a dialog with title, message, and action buttons. <br>[Guide ](./nui-components/AlertDialog.md) / [API Documentation](https://docs.tizen.org/application/dotnet/api/TizenFX/API9/api/Tizen.NUI.Components.AlertDialog.html)  |  <img src="./screenshots/AlertDialog.png" width="600"> <br>[C# code for this page](./codeExamples/AlertDialog.cs) / [XAML page](./codeExamples/AlertDialog.xaml)                |
```

Comment:
This is strange behaviour becasue all files are located in right place - I have double checked.
For example: 
**https://docs1.stg.tizen.org/staging/1522/application/dotnet/guides/user-interface/nui/codeExamples/AlertDialog.cs** - visible
**https://docs1.stg.tizen.org/staging/1522/application/dotnet/guides/user-interface/nui/codeExamples/AlertDialog.xaml** - not visible

While if you select "Files changed tab in this Pull request you can see two first following files added are"
docs/application/dotnet/guides/user-interface/nui/codeExamples/AlertDialog.cs 
docs/application/dotnet/guides/user-interface/nui/codeExamples/AlertDialog.xaml

Links to xaml files are somhow blocked here?
Redefining xaml to xml but this is not the point here.
What do you think I should do ?

Reply:
@mijin85cho , could you please help to fix the issue for XAML links in this file?

Reply:
I checked the supported file type : @patrykka is right, xaml format is not supported in docs.tizen.org.

xml is supported. Also, I asked to site manager to add xaml format to the supported list, but not sure when it will be added.

Reply:
@mijin85cho @sagarkedari  is there any update related to XAML files support?